app/initializers/irene.js

```diff
@@ -16,6 +16,40 @@ const initialize = function(application) {
   // Inject Store
   application.inject('component', 'store', 'service:store');
 
+  let runtimeConfig = window.runtimeGlobalConfig;
+
+  function parseUrl(url) {
+    let parser = document.createElement('a'), searchObject = {}, queries, split, i;
+    parser.href = url;
+    queries = parser.search.replace(/^\?/, '').split('&');
+    for(i = 0; i < queries.length; i++ ) {
+        split = queries[i].split('=');
+        searchObject[split[0]] = split[1];
+    }
+    return parser;
+  }
+
+  if(runtimeConfig) {
+    ENV.host = runtimeConfig.IRENE_API_HOST || ENV.host;
+    var devicefarmEnv = runtimeConfig.IRENE_DEVICEFARM_URL;
+    if(devicefarmEnv) {
+      let deviceFarmWebsockifyHost = parseUrl(devicefarmEnv);
+      let deviceFarmSsl = deviceFarmWebsockifyHost.protocol === "wss:";
+      ENV.deviceFarmSsl = deviceFarmSsl;
+      ENV.deviceFarmPort = deviceFarmWebsockifyHost.port || (deviceFarmSsl ? 443:80);
+      ENV.deviceFarmHost = deviceFarmWebsockifyHost.hostname;
+    }
+    ENV.socketPath = runtimeConfig.IRENE_API_SOCKET_PATH || ENV.socketPath;
+    ENV.enableSSO = runtimeConfig.IRENE_ENABLE_SSO || ENV.enableSSO;
+    ENV.isEnterprise = runtimeConfig.ENTERPRISE || ENV.isEnterprise;
+    ENV.whitelabel = {
```

Comment:
Probably should be done via Object.assign. The whitelabel object may have other params which are not runtime derived, something like `ENV.whitelabel = Object.assign({}, ENV.whitelabel, <override object>)`.

Or use es6 object spread to do the same.

Reply:
Yeah, this works if we have the same key. But, here in our case - for example if white-label is enabled then it will be `ENV.whitelabel.enabled` and key in `runtimeConfig` will be `untimeConfig.WHITELABEL_ENABLED`

Reply:
if the key can be same, then we can do that 👍

Reply:
@cosmosgenius 🔝

Reply:
the override object would be something like 
```
{
	"enabled": runtimeConfig.WHITELABEL_ENABLED,
	.
	.
	.
}
```